### PR TITLE
Fine-tuning candidate bar layout

### PR DESCRIPTION
**Pull request by graphemecluster**
### Before
Screenshot

### After
Screenshot

### CODE REVIEW

File: CantoboardFramework/Keyboard/View/LayoutConstants.swift [hunk 1]
```diff
@@ -343,9 +343,9 @@ class LayoutConstants: Copyable {
     var numOfSingleCharCandidateInRow: Int {
         switch idiom {
         case .phone:
-            return isPortrait ? 8 : 15
+            return isPortrait ? 10 : 18
```
Comment (alex-the-man) [inline CantoboardFramework/Keyboard/View/LayoutConstants.swift:346]: Could we keep this unchanged? A lot of users are complaining the candidates are too small already

File: CantoboardFramework/Keyboard/View/LayoutConstants.swift [hunk 2]
```diff
@@ -343,9 +343,9 @@ class LayoutConstants: Copyable {
...
         case .pad:
-            return isPortrait ? 15 : 20
+            return isPortrait ? 16 : 22
```
Comment (alex-the-man) [inline CantoboardFramework/Keyboard/View/LayoutConstants.swift:348]: Could we test:

1. iPad mini 7.9"
2. iPad Pro 12.9"

**Comment by graphemecluster** [comment]
OK, let’s keep it as is for the time being.

**Comment by graphemecluster** [comment]
If you don’t mind, please help me revert it.

**Comment by alex-the-man** [comment]
This doesn't look ideal for iPad either. We should come up with a responsive way to calculate the min size:
Simulator Screen Shot - iPad mini (6th generation) - 2022-02-05 at 18 32 12
Simulator Screen Shot - iPad Pro (12 9-inch) (5th generation) - 2022-02-05 at 18 32 11

**Pull request merged**